semcore/data-table/src/Body.tsx

```diff
@@ -346,12 +398,13 @@ class Body extends Component<AsProps, State> {
             ref={forkRef(...scrollContainerRefs)}
             role='rowgroup'
             focusRingTopOffset={'3px'}
+            tabIndex={-1}
           >
             {body}
           </ScrollArea.Container>
-          <div style={displayContents} role='rowgroup'>
-            <div style={displayContents} role='row'>
```

Comment:
I see that you have removed row role. Does it still passes a11y checks?

Reply:
Yes, I checked that